PyISY/Nodes/node.py

```diff
@@ -154,3 +155,24 @@ def dim(self):
             self.parent.parent.log.info('ISY dimmed node: ' + self._id)
             self.update(_change2update_interval)
             return True
+
+    @property
+    def spoken(self):
+        if self._spoken is None:
+            # Get the device notes, currently only the Spoken property is saved.
+            notes_xml = self.parent.parent.conn.getNodeNotes(self._id)
+            spoken = False
+            if notes_xml is not None:
+                try:
+                    notesdom = minidom.parseString(notes_xml)
+                except:
+                    self.parent.log.error('ISY Could not parse node ' + self._id + ' notes '
+                                          + 'poorly formatted XML.')
+                spoken_tag = notesdom.getElementsByTagName('spoken')
+                if len(spoken_tag) > 0:
```

Comment:
That doesn't result in the same logic.   The __init__ sets _spoken to None which means it hasn't been checked yet.  Then when it's checked and is not defined or null, spoken method sets it to False.  Changing the log would result in it being set to None when the spoken tag exists, but is empty.  So I can change the logic so _init__ initializes it to False, and once it's checked set to None when not defined or null.